src/commands/createNewProject/CSharpProjectCreator.ts

```diff
@@ -57,11 +54,11 @@ export class CSharpProjectCreator implements IProjectCreator {
             throw new Error(localize('unrecognizedTargetFramework', 'Unrecognized target framework in project file "{0}".', csProjName));
         } else {
             const targetFramework: string = matches[1];
```

Comment:
Why matches[1] and not [0] out of curiosity?

Reply:
matches[0] is always the _full_ match and the rest of the matches in a RegExpMatchArray are from the capturing groups. So this case:
```
console.log(matches[0]); // Prints "<TargetFramework>net461<\/TargetFramework>"
console.log(matches[1]); // Prints "net461"
```